Mathlib/Algebra/Algebra/Equiv.lean

```diff
@@ -787,7 +787,20 @@ variable {M G : Type*} (R A : Type*) [CommSemiring R] [Semiring A] [Algebra R A]
 
 section
 
-variable [Group G] [MulSemiringAction G A] [SMulCommClass G R A]
+variable [Group G]
+
+/-- Construct a `MulSemiringAction` from a homomorphism `G →* A ≃ₐ[R] A`. -/
```

Comment:
I'm not sure what the precedence is between `→*` and `≃ₐ[R]`. Can you parenthesise the comment, at least?